biz.aQute.bnd/src/aQute/bnd/main/RemoteCommand.java

```diff
@@ -109,6 +111,57 @@ public void event(Event e) throws Exception {
 		agent = launcher.getAgent();
 	}
 
+	@Description("List the bundles already installed in remote framework")
+	@Arguments(arg = {})
+	interface ListOptions extends Options {
+	}
+
+	public void _list(ListOptions options) throws Exception {
+		List<BundleDTO> installedBundles = agent.getBundles();
+		installedBundles.forEach(this::prettyPrintBundleDTO);
+	}
+
+	@Description("Communicate with the remote agent to install or update bundle")
+	interface InstallOptions extends Options {
+		@Description("Specify the location identifier of the bundle")
```

Comment:
I do not think the location and file should be options. Just make them the first and second argument?

It is generally also nice if the second argument (the file) can also be a URL

Reply:
Done